avaframe/com1DFA/com1DFACfg.ini

```diff
@@ -28,3 +26,14 @@ aimecDir = Outputs/com1DFA/
 calcPressureLimit = 10
 # width of the avalanche path domain for computations in Aimec
 domainWidth = 600
```

Comment:
This is not needed anymore. It is set in the AIMEC.ini file
You don't even need to write this .txt file anymore

Reply:
Ok, now there is no section Aimec anymore in the .ini